lms/djangoapps/program_enrollments/api/linking.py

```diff
@@ -169,6 +170,7 @@ def link_program_enrollment_to_lms_user(program_enrollment, user):
                 user, pce.course_key, pce.status
             )
             pce.save()
+            _realize_course_access_roles(user, pce)
```

Comment:
Is "realize" still the word we use when we talk about "matriculation" of a learner? Or is that no longer the term?

Reply:
The thesaraus online gives many choice based on `realize`. See https://www.thesaurus.com/browse/realize

I personally like the word `fulfill`